frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePickerHeader.styled.tsx

```diff
@@ -23,16 +22,17 @@ export const TabButton = styled(Button)<TabButtonProps>`
   padding-right: 0;
   margin-left: ${space(2)};
   margin-right: ${space(2)};
-  border-bottom: ${({ primaryColor = "var(--mb-color-brand)", selected }) =>
-    selected ? `2px solid ${primaryColor}` : "2px solid transparent"};
-  color: ${({ primaryColor = "var(--mb-color-brand)", selected }) =>
-    selected ? primaryColor : color("text-medium")};
+  border-bottom: ${({ selected }) =>
```

Comment:
the change is basically removing `primaryColor` and replace it with `var(--mb-color-text-brand)`